app/src/main/res/layout/fragment_plant_detail.xml

```diff
@@ -14,166 +14,8 @@
-                    app:layout_constraintEnd_toEndOf="parent"
-                    app:layout_constraintStart_toStartOf="parent"
-                    app:layout_constraintTop_toBottomOf="@id/plant_detail_name" />
-
-                <TextView
-                    android:id="@+id/plant_watering"
-                    android:layout_width="0dp"
-                    android:layout_height="wrap_content"
-                    android:layout_marginStart="@dimen/margin_small"
-                    android:layout_marginEnd="@dimen/margin_small"
-                    android:gravity="center_horizontal"
-                    app:layout_constraintEnd_toEndOf="parent"
-                    app:layout_constraintStart_toStartOf="parent"
-                    app:layout_constraintTop_toBottomOf="@id/plant_watering_header"
-                    app:wateringText="@{viewModel.plant.wateringInterval}"
-                    tools:text="every 7 days" />
-
-                <TextView
-                    android:id="@+id/plant_description"
-                    style="?android:attr/textAppearanceMedium"
-                    android:layout_width="0dp"
-                    android:layout_height="wrap_content"
-                    android:layout_marginStart="@dimen/margin_small"
-                    android:layout_marginTop="@dimen/margin_small"
-                    android:layout_marginEnd="@dimen/margin_small"
-                    android:textIsSelectable="true"
-                    android:minHeight="@dimen/plant_description_min_height"
-                    app:layout_constraintEnd_toEndOf="parent"
-                    app:layout_constraintStart_toStartOf="parent"
-                    app:layout_constraintTop_toBottomOf="@id/plant_watering"
-                    app:renderHtml="@{viewModel.plant.description}"
-                    tools:text="Details about the plant" />
-
-            </androidx.constraintlayout.widget.ConstraintLayout>
-
-        </androidx.core.widget.NestedScrollView>
-
-        <com.google.android.material.floatingactionbutton.FloatingActionButton
-            android:id="@+id/fab"
-            style="@style/Widget.MaterialComponents.FloatingActionButton"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:layout_margin="@dimen/fab_margin"
-            android:onClick="@{() -> callback.add(viewModel.plant)}"
-            android:tint="@android:color/white"
-            app:shapeAppearance="@style/ShapeAppearance.Sunflower.FAB"
-            app:isGone="@{viewModel.isPlanted}"
-            app:layout_anchor="@id/appbar"
-            app:layout_anchorGravity="bottom|end"
-            app:srcCompat="@drawable/ic_plus" />
-
-    </androidx.coordinatorlayout.widget.CoordinatorLayout>
-
-</layout>
\ No newline at end of file
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/compose_frame"
+    android:background="?attr/colorSurface"
```

Comment:
Have both a background on the view and a `Surface` at the compose root. Do you need both?